ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__mce.yaml

```diff
@@ -45,7 +45,6 @@ tests:
         OSCI_PUBLISH_DELAY=0
         OSCI_PIPELINE_PRODUCT_PREFIX=backplane
         OSCI_PIPELINE_REPO=backplane-pipeline
-      RELEASE_REF: backplane-2.4
```

Comment:
Why? I mean what's suppose to happen without this value?

Reply:
This value was initially set due to needing to override your branch value in the pipeline and identify where to place it, that shouldn't be required anymore and there have been updates to better track where component updates are coming from so removing the override acts as data sanitation.

Reply:
thanks for the explanation
please have that in the PR description and commit message as future documentation to this change